evergreen.yml

```diff
@@ -1766,8 +1853,13 @@ tasks:
       - func: "generate SBOM"
       - func: "scan SBOM"
       - func: "push SBOM Lite to Silk"
-      - func: "pull augmented SBOM from Silk"
-      - func: "publish augmented SBOM"
+
+  - name: compliance-report
```

Comment:
I don't think that you need a separate task for this one though.
Call the same function in both the -release and -snapshot task